services/movie.py

```diff
@@ -0,0 +1,44 @@
+from django.db.models import QuerySet
+from db.models import Movie
+
+
+def get_movies(
+        genres_ids: list[int] = None,
+        actors_ids: list[int] = None
+) -> QuerySet:
+    queryset = Movie.objects.all()
+    if genres_ids is None and actors_ids is None:
+        return queryset
+    elif genres_ids is not None and actors_ids is not None:
+        return queryset.filter(
+            genres__id__in=genres_ids, actors__id__in=actors_ids
+        )
+    elif genres_ids is not None and actors_ids is None:
+        return queryset.filter(genres__id__in=genres_ids)
+    elif genres_ids is None and actors_ids is not None:
+        return queryset.filter(actors__id__in=actors_ids)
```

Comment:
Lots of redundant checks
No need to return queryset immediately, you can reassign it to a new value
for example, if genres_ids was passed, then you can do
`queryset = queryset.filter(genres__id__in=genres_ids)`

The same for actors

Reply:
redone